fio_wrapper/ci_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -x
+
+source ci/common.sh
+
+# Build image for ci
+podman build --tag=quay.io/cloud-bulldozer/fio:snafu_ci -f fio_wrapper/Dockerfile . && podman push quay.io/cloud-bulldozer/fio:snafu_ci
+
+cd ripsaw
+
+sed -i 's/fio:latest/fio:snafu_ci/g' roles/fio-distributed/templates/*
+
+# Build new ripsaw image
+update_operator_image snafu_ci
+
+sed -i '/trap finish EXIT/d' tests/test_fiod.sh
+
+rm -f uuid
+(
+source tests/test_fiod.sh || : 
+
+# Get UUID
+uuid=`kubectl -n my-ripsaw get benchmarks -o jsonpath='{.items[0].status.uuid}'`
+
+finish
+echo $uuid > uuid
+)
+uuid=`cat uuid`
+
+cd ..
+
+# Define ES server info (this is the same from the test cr
+es_server="marquez.perf.lab.eng.rdu2.redhat.com"
+es_port=9200
+index="ripsaw-fio-results"
```

Comment:
we'll need to also look at `ripsaw-fio-logs` and `ripsaw-fio-analyzed_result` for full coverage.